README.md

```diff
@@ -28,68 +28,23 @@ Features not supported:
-- `fromV1(input, password)` - import a wallet (Version 1 of the Ethereum wallet format)
-- `fromV3(input, password, [nonStrict])` - import a wallet (Version 3 of the Ethereum wallet format). Set `nonStrict` true to accept files with mixed-caps.
-- `fromEthSale(input, password)` - import an Ethereum Pre Sale wallet
-
-For the V1, V3 and EthSale formats the input is a JSON serialized string. All these formats require a password.
-
-Note: `fromPublicKey()` only accepts uncompressed Ethereum-style public keys, unless the `nonStrict` flag is set to true.
-
-Instance methods:
-
-- `getPrivateKey()` - return the private key
-- `getPublicKey()` - return the public key
-- `getAddress()` - return the address
-- `getChecksumAddressString()` - return the [address with checksum](https://github.com/ethereum/EIPs/issues/55)
-- `getV3Filename([timestamp])` - return the suggested filename for V3 keystores
-- `toV3(password, [options])` - return the wallet as a JSON string (Version 3 of the Ethereum wallet format)
-
-All of the above instance methods return a Buffer or JSON. Use the `String` suffixed versions for a string output, such as `getPrivateKeyString()`.
-
-Note: `getPublicKey()` only returns uncompressed Ethereum-style public keys.
+For information about the Wallet's API, please go to [./docs/classes/wallet.md](./docs/classes/wallet.md).
 
 ## Thirdparty API
 
 Importing various third party wallets is possible through the `thirdparty` submodule:
 
-`var thirdparty = require('ethereumjs-wallet/thirdparty')`
-
-Constructors:
+`const thirdparty = require('ethereumjs-wallet/thirdparty')`
 
-- `fromEtherCamp(passphrase)` - import a brain wallet used by Ether.Camp
-- `fromEtherWallet(input, password)` - import a wallet generated by EtherWallet
-- `fromKryptoKit(seed)` - import a wallet from a KryptoKit seed
-- `fromQuorumWallet(passphrase, userid)` - import a brain wallet used by Quorum Wallet
+Please go to [./docs/README.md](./docs/README.md) for more info.
 
 ## HD Wallet API
 
 To use BIP32 HD wallets, first include the `hdkey` submodule:
 
-`var hdkey = require('ethereumjs-wallet/hdkey')`
-
-Constructors:
-
-- `fromMasterSeed(seed)` - create an instance based on a seed
-- `fromExtendedKey(key)` - create an instance based on a BIP32 extended private or public key
-
-For the seed we suggest to use [bip39](https://npmjs.org/package/bip39) to create one from a BIP39 mnemonic.
-
-Instance methods:
+`const hdkey = require('ethereumjs-wallet/hdkey')`
 
-- `privateExtendedKey()` - return a BIP32 extended private key (xprv)
-- `publicExtendedKey()` - return a BIP32 extended public key (xpub)
-- `derivePath(path)` - derive a node based on a path (e.g. m/44'/0'/0/1)
-- `deriveChild(index)` - derive a node based on a child index
-- `getWallet()` - return a `Wallet` instance as seen above
+Please go to [./docs/classes/ethereumhdkey.md](./docs/classes/ethereumhdkey.md) for more info.
 
 ## Provider Engine
```

Comment:
Can you add a link here in the section to the generated `WalletSubprovider` class doc?

Reply:
Added it